src/containerapp/azext_containerapp/_transformers.py

```diff
@@ -17,6 +17,26 @@ def transform_containerapp_output(app):
     return result
 
 
+def clean_up_sensitive_values(response_json):
+    for container in safe_get(response_json, "properties", "template", "containers", default=[]):
+        if "env" in container:
+            for env in container["env"]:
+                if env.get("value"):
+                    del env["value"]
+
+    if safe_get(response_json, "properties", "template") and "scale" in response_json["properties"]["template"]:
+        for rule in safe_get(response_json, "properties", "template", "scale", "rules", default=[]):
+            for (key, val) in rule.items():
+                if key != "name":
+                    val["metadata"] = dict((k, "") for k, v in val["metadata"].items())
+
+    return response_json
+
+
+def transform_sensitive_values_wrapper():
+    return clean_up_sensitive_values
```

Comment:
I suggest merging the two methods `clean_up_sensitive_values `and `transform_sensitive_values_wrapper `together.
Seems no need to separate them.